### PR TITLE
Vet Clinic: performance audit PR

### PR DESCRIPTION
**Pull request by federicaulzurrun**
### In this project @flemton and I:

- Added the column email to the owner's table as required.
- Added data to the database with the commands given by Microvers.
- Added improvements so that when we run the EXPLAIN ANALYZE the execution time is less than 1000.

### Screenshots:

Execution time before improvements :

1. 
animalMore

2. 
vetMore

3. 
emailMore

Execution time after improvements:

1.  
animalLess

2. 
vetLess

3. 
emailLess

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -94,3 +94,25 @@ INSERT INTO visits (animal_id, vet_id, date_of_visit)
...
+CREATE INDEX idx_visits_animal_id ON visits (animal_id);
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+
+
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+CREATE INDEX idx_visits_vet_id ON visits (vet_id);
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
+CREATE INDEX idx_owners_email ON owners (email);
```
Comment (DuaneDave) [inline data.sql]: Good job resolving the n + 1 problem with the database, however, the commands that create an index on the table should be in the schema.sql file. Consider doing so to correctly separate db set up from db operation

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @federicaulzurrun and @flemton  👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good job handling n + 1 problem
✔️ Your query execution time seem lightening fast


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (federicaulzurrun) [reply data.sql]: @DuaneDave Thanks! one question, just the create index of the three of the right? not with the explain analyze?

**Review by Reem-lab — APPROVED** [review]
# Hi Team,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Great job on the funtionality 💯
- [ ] Followed Gitflow 🔥
- [ ] No Linters error ✅
- [ ] Kudos wokring on the requested changes⭐


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**